test/dag_benchmarks.cc

```diff
@@ -135,12 +135,25 @@ TEST_CASE("benchmark task handling", "[benchmark][group:task-graph]") {
 }
 
 
+static void set_benchmark_task_manager_policies(task_manager& tm) {
+	// these policies are equivalent to the ones used by `runtime` (except that we throw exceptions here for benchmark-debugging purposes)
+	tm.set_uninitialized_read_policy(CELERITY_ACCESS_PATTERN_DIAGNOSTICS ? error_policy::throw_exception : error_policy::ignore);
+}
+
+static void set_benchmark_command_graph_generator_policies(distributed_graph_generator& dggen) {
+	// these policies are equivalent to the ones used by `runtime` (except that we throw exceptions here for benchmark-debugging purposes)
+	dggen.set_uninitialized_read_policy(error_policy::ignore); // uninitialized reads already detected by task manager
+	dggen.set_overlapping_write_policy(CELERITY_ACCESS_PATTERN_DIAGNOSTICS ? error_policy::throw_exception : error_policy::ignore);
+}
+
+
```

Comment:
I forgot to configure benchmark graph generation before, now it should match what the runtime does.